paymentsheet/src/main/java/com/stripe/android/paymentsheet/forms/PaypalSpec.kt

```diff
@@ -0,0 +1,25 @@
+package com.stripe.android.paymentsheet.forms
+
+import com.stripe.android.paymentsheet.elements.LayoutSpec
+
+/**
+ * This defines the requirements for usage as a Payment Method.
+ */
+internal val PaypalRequirement = PaymentMethodRequirements(
+    piRequirements = emptySet(),
+
+    /**
+     * SetupIntents are not supported by this payment method. Currently Paypal should only be set up
+     * one time, but the Stripe API will let you keep doing it resulting in several paypals being
+     * listed. It's better to leave this disabled than to enable it, let users save multiple
+     * paypals, and then fix it leaving the users in a weird state. See oof #5
```

Comment:
I would like to make sure that this comment is properly reflected in the oof doc before closing this one out.

As I understand it there might be a couple of issues:
1. You can add a PayPal PM to the same customer multiple times.
2. PayPal does not work on_session, which is the scenario when selecting a saved customer card.  The user is present = on_session.
3. A PayPal PM attached to a customer cannot be used to confirm a SI or PI + SFU (not sure if the SFU value of on or off-session matter here).